og_access/src/Plugin/OgFields/OgAccessField.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+namespace Drupal\og_access\Plugin\OgFields;
+
+use Drupal\og\OgFieldBase;
+use Drupal\og\OgFieldsInterface;
+use Drupal\og_access\OgAccess;
+
+/**
+ * Determine if group should use default roles and permissions.
+ *
+ * @OgFields(
+ *  id = OG_ACCESS_FIELD,
```

Comment:
```suggestion
 *  id = \Drupal\og_access\OgAccess\OgAccess\OG_ACCESS_FIELD,
```

Uncaught PHP Exception Doctrine\\Common\\Annotations\\AnnotationException: "[Semantical Error] Couldn't find constant OG_ACCESS_FIELD